packages/test/local-server-stress-tests/package.json

```diff
@@ -0,0 +1,118 @@
+{
+	"name": "@fluid-internal/local-server-stress-tests",
+	"version": "2.23.0",
+	"private": true,
+	"description": "Stress tests that can only run against the local server",
+	"homepage": "https://fluidframework.com",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/microsoft/FluidFramework.git",
+		"directory": "packages/test/local-server-stress-tests"
+	},
+	"license": "MIT",
+	"author": "Microsoft and contributors",
+	"sideEffects": false,
+	"type": "commonjs",
+	"scripts": {
+		"build": "fluid-build . --task build",
+		"build:compile": "fluid-build . --task compile",
+		"build:test": "tsc --project ./src/test/tsconfig.json",
+		"check:biome": "biome check .",
+		"check:format": "npm run check:biome",
+		"check:prettier": "prettier --check . --cache --ignore-path ../../../.prettierignore",
+		"clean": "rimraf --glob dist lib \"**/*.tsbuildinfo\" \"**/*.build.log\" nyc",
+		"eslint": "eslint --format stylish src",
+		"eslint:fix": "eslint --format stylish src --fix --fix-type problem,suggestion,layout",
+		"format": "npm run format:biome",
+		"format:biome": "biome check . --write",
+		"format:prettier": "prettier --write . --cache --ignore-path ../../../.prettierignore",
+		"lint": "fluid-build . --task lint",
+		"lint:fix": "fluid-build . --task eslint:fix --task format",
+		"test": "npm run test:mocha",
+		"test:coverage": "c8 npm test",
+		"test:mocha": "mocha \"lib/test/**/*.spec.*js\"  --exit",
```

Comment:
Is `--exit` actually necessary? OK if so, but if this is just copy-pasted from another package.json, I'd expect it to not be necessary and we have some other scheduled work to remove this flag, so would be good to avoid propagating usage.

Reply:
to judge "if necessary", the places where we have issues and need to pass `--exit` cause the test running process to hang if `--exit` is not passed.

I guess it's plausible you do actually need it due to local server usage, but wanted to double-check.